_includes/docs_menu.html

```diff
@@ -23,6 +23,7 @@
     <li><a href="{{ site.baseurl }}/docs/auth-modes">Authentication Modes</a></li>
     <li><a href="{{ site.baseurl }}/docs/log-entries">Signature Verification Log Entries</a></li>
     <li><a href="{{ site.baseurl }}/docs/gallery-submission">Gallery Submission Guide</a></li>
+    <li><a href="{{ site.baseurl }}/docs/metadata-enumeration">Metadata Enumeration</a></li>
```

Comment:
Can you put this between capabilities and auth modes, since it seems like this list is sort of sorted by importance.